resources/js/components/felte-form/FelteSubmit.tsx

```diff
@@ -31,7 +31,7 @@ export const FelteSubmit: ParentComponent<Props> = (props) => {
       <div class="flex gap-1 justify-center items-stretch">
         <Show when={props.cancel}>
           <Button
-            class="flex-grow basis-0 border border-cyan-500 disabled:bg-opacity-50 rounded p-2 text-cyan-500"
+            class="flex-grow basis-0 secondary"
```

Comment:
Zazwyczaj warianty Buttona robi się za pomocą propa variant, np. variant="primary" albo variant="secondary", ma to swoją zaletę - większe type-safety i VSCode podpowiada.

Ale poza tym nie widzę przeciwswskazań w tym rozwiązaniu i moim zdaniem może tak zostać. Miłe urozmaicenie i zdaje się lżejsze niż dodawanie nowego propa.

Reply:
Hm, to chyba zostawię po prostu te klasy. Zawsze można kiedyś później zmienić gdybyśmy uznali że trzeba.